### PR TITLE
Update Java snippets for twilio-java 7.17

### PR DESCRIPTION
**Pull request by joliveros**

### CODE REVIEW

File: sync/rest/maps/update-map-item/update-map-item.7.x.java [hunk 1]
```diff
@@ -17,12 +16,14 @@
   public static void main(String[] args) {
     Twilio.init(ACCOUNT_SID, AUTH_TOKEN);
 
-    Map<String, Object> data = new HashMap<>();
+    HashMap<String, Object> data = new HashMap<String, Object>();
```
Comment (hortega) [inline sync/rest/maps/update-map-item/update-map-item.7.x.java]: It's good practice to leave the parent type on the left.  the generic types are inferred on the right side, so they are not needed. You can write:
`Map<String, Object> data = new HashMap<>();`

File: rest/taskrouter/twiml/example2/example/example.7.x.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import com.twilio.twiml.EnqueueTask;
+import com.twilio.twiml.voice.EnqueueTask;
```
Comment (efossier) [inline rest/taskrouter/twiml/example2/example/example.7.x.java]: EnqueueTask no longer exists. It was a special case of Enqueue. See https://docs.google.com/document/d/1WexET5KxnlI4i6i6gH1SUytYnYtkQZHUF8dGz2DI-IE/edit for migrating.

File: rest/taskrouter/twiml/example3/example/example.7.x.java [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import com.twilio.twiml.EnqueueTask;
+import com.twilio.twiml.voice.EnqueueTask;
```
Comment (efossier) [inline rest/taskrouter/twiml/example3/example/example.7.x.java]: ditto

File: rest/taskrouter/twiml/example4/example/example.7.x.java [hunk 4]
```diff
@@ -5,7 +5,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import com.twilio.twiml.EnqueueTask;
+import com.twilio.twiml.voice.EnqueueTask;
```
Comment (efossier) [inline rest/taskrouter/twiml/example4/example/example.7.x.java]: ditto

File: twiml/voice/enqueue/enqueue-1/enqueue-1.7.x.java [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import com.twilio.twiml.Enqueue;
+import com.twilio.twiml.voice.Enqueue;
```
Comment (efossier) [inline twiml/voice/enqueue/enqueue-1/enqueue-1.7.x.java]: `queueName` in the code below needs to be renamed to `name`

File: twiml/voice/leave/leave-1/leave-1.7.x.java [hunk 6]
```diff
@@ -1,6 +1,6 @@
-import com.twilio.twiml.Enqueue;
+import com.twilio.twiml.voice.Enqueue;
```
Comment (efossier) [inline twiml/voice/leave/leave-1/leave-1.7.x.java]: `queueName` in code below needs to be changed to `name`

**Pull request merged**